services/core/shared/core/compat/sdk_v5/accounts.js

```diff
@@ -161,7 +161,7 @@ const indexAccountsbyAddress = async (addressesToIndex, isGenesisBlockAccount =
 
 			// A genesis block account is considered migrated
 			if (isGenesisBlockAccount) await isGenesisAccountCache.set(address, true);
-			const accountFromDB = await getIndexedAccountInfo({ address });
+			const accountFromDB = await getIndexedAccountInfo({ address, limit: 1 }, 'publicKey');
```

Comment:
Although an optional change, it would be nice if we always pass the columns within an array. This would keep the implementation consistent across the application.
```suggestion
			const accountFromDB = await getIndexedAccountInfo({ address, limit: 1 }, ['publicKey']);
```